.github/workflows/pythonpackage.yml

```diff
@@ -15,51 +15,70 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9, "3.10", "3.11"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
I agree that there is no need to claim lack of support for python 3.7 as long as we are still compatible with it. Maybe py3.7 support could be re-added to GitDB as well for clarity.